UnitTests/GitUITests/UserControls/ToolStripPushButtonTests.cs

```diff
@@ -44,6 +44,8 @@ public void Setup()
         [Test]
         public void DisplayAheadBehindInformation_should_not_display_anything_if_does_not_support_ahead_behind()
         {
+            AppSettings.ShowAheadBehindData = true;
```

Comment:
We shouldn't be doing this as it modifies a global shared state, and introduces a risk of breaking some other tests that may depend on this.